test/common/event/dispatched_thread_impl_test.cc

```diff
@@ -0,0 +1,46 @@
+#include <functional>
+
+#include "common/common/utility.h"
+#include "common/event/dispatched_thread.h"
+
+#include "server/guarddog_impl.h"
+
+#include "test/mocks/common.h"
+#include "test/mocks/server/mocks.h"
+#include "test/mocks/stats/mocks.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+using testing::InSequence;
```

Comment:
nit: move outside of Envoy namespace (this got broken in the reformat, have been trying to fix)